File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        int sum = Integer.parseInt("" + a + b);
        int mul = 2 * a * b;
        
        if(sum > mul || sum == mul) {
            return sum;
        }else {
            return mul;
        }
    }
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.py
def solution(a, b):

    ab = int(str(a) + str(b))
    ab2 = 2 * a * b
    
    return max(ab, ab2)

# 다른 풀이
return int(str(a) + str(b)) if int(str(a) + str(b)) > 2*a*b else 2*a*b
